### PR TITLE
Fix/no ref/selected message shadow

### PR DESCRIPTION
**Pull request by yougotwill**
Fixes the shadow effect when in multi-select mode for all message types

### CODE REVIEW

File: ts/components/conversation/message/message-item/GenericReadableMessage.tsx [hunk 1]
```diff
@@ -118,11 +119,16 @@ const StyledReadableMessage = styled(ReadableMessage)<{
   width: 100%;
   letter-spacing: 0.03rem;
   padding: var(--margins-xs) var(--margins-lg) 0;
+  margin: var(--margins-xxs) 0;
```
Comment (Bilb) [inline ts/components/conversation/message/message-item/GenericReadableMessage.tsx:122]: there is no way to avoid that 2.5px margin here? This looks kind of magic. 
Why is that needed?

Comment (yougotwill) [reply ts/components/conversation/message/message-item/GenericReadableMessage.tsx:122]: It's a bit hard to spot but it's about the spacing between attachments under a message (or a voice message) and the next message or date break. I needed a smaller unit of measure to get the spacing to match. If you use `margin-xs` it is slightly too big and the spacing between different message elements starts to look too large and uneven. If you want I can reference the 2.5px directly in the `GenericReadableMessage` component since it is only used in one place?

Comment (Bilb) [reply ts/components/conversation/message/message-item/GenericReadableMessage.tsx:122]: :+1:
